src/main/java/org/colonelkai/publictransit/TravelManager.java

```diff
@@ -0,0 +1,115 @@
+package org.colonelkai.publictransit;
+
+import org.colonelkai.publictransit.line.travel.Travel;
+import org.core.TranslateCore;
+import org.core.entity.living.human.player.User;
+import org.core.schedule.Scheduler;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+public class TravelManager {
+    private final HashMap<Travel, Scheduler> travelMap = new HashMap<>();
+
+    private final Consumer<Scheduler> consumer = (Scheduler s) -> {
```

Comment:
Single character names is bad practise in any language. Please use ``scheduler`` or some other full word(s) to describe what it is. Exception being if you dont use it